### PR TITLE
Ray_on_Spark Parameter_server Update

### PR DESCRIPTION
**Pull request by sgwhat**
This PR is created to solve some problems caused by importing way we did original, just move the parameter_sever model into python/orca/src/bigdl/orca/ray

### CODE REVIEW

**Comment by sgwhat** [comment]
Test passed.

File: python/orca/src/bigdl/orca/ray/model.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
+# This file is adapted from https://github.com/ray-project/ray/blob
```
Comment (jason-dai) [inline python/orca/src/bigdl/orca/ray/model.py:1]: this should be an example, not part of the main library

Comment (sgwhat) [reply python/orca/src/bigdl/orca/ray/model.py:1]: Yeah, but the original way to import model in async_parameter_server.py is not working in yarn-test and k8s-test, put model.py in src could solve problems we met before.

Comment (jason-dai) [reply python/orca/src/bigdl/orca/ray/model.py:1]: "put model.py in src" is not the correct solution; what if the users need to build their own code on yarn or k8s? They cannot put their code in src.

Comment (sgwhat) [reply python/orca/src/bigdl/orca/ray/model.py:1]: yes, I'll think about other solutions.

**Comment by sgwhat** [comment]
Jenkins passed separately in http://10.112.231.51:18888/view/ZOO-PR/job/ZOO-PR-Python-integration-test/401/ and http://10.112.231.51:18888/view/ZOO-PR/job/ZOO-PR-Python-integration-test/408/

**Pull request merged**